docs/dheg-build.gradle

```diff
@@ -34,7 +34,8 @@ buildscript {
 apply plugin: "info.solidsoft.pitest"
 
 pitest {
-  mutationEngine = "descartes"
+  targetClasses = ['myWorld*']
+//  mutationEngine = "descartes"
```

Comment:
Wouldn't this mean that descartes is not used?

Reply:
Yes, you are right, but I get errors if I leave that line. 
Do you know how to modify it to run pitest using gradle?

Reply:
Ok, I found another error in the tutorial. I will update it and push it.